code/game/atoms_movable.dm

```diff
@@ -299,7 +299,7 @@
 		pulledby.stop_pulling()
 
 	// They are moving! Wouldn't it be cool if we calculated their momentum and added it to the throw?
-	if(thrower && thrower.last_move && thrower.client && thrower.client.move_delay >= world.time + world.tick_lag * 2)
+	if(thrower && istype(thrower) && thrower.last_move && thrower.client && thrower.client.move_delay >= world.time + world.tick_lag * 2)

```

Comment:
This is the worst way to fix things. This means you can intentionally pass a poor argument and never know it's bad.

When things are used improperly or a wrong argument is passed that typically only should have one type of thing, then we *want* it to runtime and fix the problem at the source.

Reply:
Took this bit out.